Assets/HoloToolkit/Input/Scripts/InputSources/GesturesInput.cs

```diff
@@ -89,80 +93,120 @@ protected override void OnDestroy()
 
                 gestureRecognizer.Dispose();
             }
+            if (navigationGestureRecognizer != null)
+            {
+                navigationGestureRecognizer.NavigationStartedEvent -= OnNavigationStartedEvent;
+                navigationGestureRecognizer.NavigationUpdatedEvent -= OnNavigationUpdatedEvent;
+                navigationGestureRecognizer.NavigationCompletedEvent -= OnNavigationCompletedEvent;
+                navigationGestureRecognizer.NavigationCanceledEvent -= OnNavigationCanceledEvent;
+
+                navigationGestureRecognizer.Dispose();
+            }
+        }
 
-            base.OnDestroy();
+        protected virtual void OnDisable()
+        {
+            StopGestureRecognizer();
+        }
+
+        protected virtual void OnEnable()
+        {
+            if (RecognizerStart == RecognizerStartBehavior.AutoStart)
+            {
+                StartGestureRecognizer();
+            }
+        }
+
+        /// <summary>
+        /// Make sure the gesture recognizer is off, then start it.
+        /// Otherwise, leave it alone because it's already in the desired state.
+        /// </summary>
+        public void StartGestureRecognizer()
```

Comment:
@DanHolbert @NeerajW That's an internal implementation detail that the user should not need to know (as this is a public method). I copied this from the KeywordManager to keep it coherent but I think these should be called simply Start and Stop. The name of the classes don't include Recognizer and there's no need to know there's one or more inside.